images/Makefile

```diff
@@ -52,7 +54,8 @@ clonerefs: docker-build-clonerefs
 ###
 
 docker-build-%:
-	docker build --build-arg IMAGE_ARG=$(REPO)/$*:$(VERSION) --build-arg VERSION=$(VERSION) -t $(REPO)/$*:$(VERSION) $*
+	-docker pull $(REPO)/$*:$(VERSION)
```

Comment:
What's this `-` before `docker pull ...`?

Reply:
The `-` tells Make to ignore the exit code, so if we can't pull the image (cause it's not on the registry) then we continue anyway